dtcInterfaceLib/DTC.cpp

```diff
@@ -1458,11 +1458,12 @@ int DTCLib::DTC::ReadBuffer(const DTC_DMA_Engine& channel, int tmo_ms)
 	}
 
 	int errorCode;
-	TRACE_EXIT { DTC_TLOG(TLVL_ReadBuffer) << "ReadBuffer found " << ((errorCode>0)?"DATA":"NO Data"); };
+	//TRACE_EXIT { DTC_TLOG(TLVL_ReadBuffer) << "ReadBuffer found " << ((errorCode>0)?"DATA":"NO Data"); };
```

Comment:
I think this TRACE_EXIT was in the correct place, it was intended to run only once on leaving scope of the entire function call. Inside the do_loop will call the message every time it loses scope in the do_loop. If we want a message every time through the loop, I think we should just have the DTC_TLOG only.

Reply:
ah! That makes sense. I reverted it. Sorry for that.